base/ocsp/src/main/java/com/netscape/cms/ocsp/LDAPStore.java

```diff
@@ -399,11 +405,11 @@ public SingleResponse processRequest(Request req) throws Exception {
         }
 
         if (theCert == null) {
-            throw new Exception("Missing issuer certificate");
+            throw new Exception("Issuer certificate not found/served in mCRL");
```

Comment:
Exception messages are meant for users, not developers, so I'd suggest not to use an internal variable name like `mCRL`.